File: practical sheet 3/code9.c
#include <stdio.h>
void main (){
	int a,b,c;
	printf("Enter value of a : ");
	scanf("%d",&a);
	printf("Enter value of b : ");
	scanf("%d",&b);
	printf("Enter value of c : ");
	scanf("%d",&c);
	if(a*a+b*b==c*c){
		printf("it is a pythagorus triplet");
	}else {
		printf("it is not a pythagorus triplet");
	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/4/code9.java
import java.util.Scanner;

class IfElse{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter a : ");
		float a = s.nextFloat();
		System.out.println("enter b : ");
		float b = s.nextFloat();
		System.out.println("enter c : ");
		float c = s.nextFloat();

		if( c*c ==(a*a+b*b) ) {

			System.out.println("a,b,c is pythogoren triplet");
		}else {

			System.out.println("a,b,c is not pythogoren triplet");

		}

	}


}
